internal/errwrap/errwrap.go

```diff
@@ -20,15 +20,8 @@ import (
 	"golang.org/x/tools/go/types/typeutil"
 )
 
-var Analyzer = &analysis.Analyzer{
-	Name:             "errwrap",
-	Doc:              "wrap errors in fmt.Errorf() calls with the %w verb directive",
-	Requires:         []*analysis.Analyzer{inspect.Analyzer},
-	Run:              run,
-	RunDespiteErrors: true,
-}
-
-func run(pass *analysis.Pass) (interface{}, error) {
+// Run is the runner for an analysis pass
+func Run(pass *analysis.Pass) (interface{}, error) {
 	inspect := pass.ResultOf[inspect.Analyzer].(*inspector.Inspector)
 
 	nodeFilter := []ast.Node{
```

Comment:
Just remove the `internal/` folder and move it to an upper level. So it should be `github.com/fatih/errwrap/errwrap`. And while you're at it, I think we can also remove the `cmd` folder. Move the `main.go` folder to the root so we can install it via `go get github.com/fatih/errwrap`.

Reply:
I didn't want to export everything but sounds cleaner. Updated.